tests/system/action/organization/test_initial_import.py

```diff
@@ -0,0 +1,531 @@
+            "default_structure_level": "",
+            "default_vote_weight": "1.000000",
+            "last_email_send": None,
+            "is_demo_user": False,
+            "organization_management_level": "superadmin",
+            "is_present_in_meeting_ids": [],
+            "comment_$": [],
+            "number_$": [],
+            "structure_level_$": [],
+            "about_me_$": [],
+            "speaker_$_ids": [],
+            "personal_note_$_ids": [],
+            "supported_motion_$_ids": [],
+            "submitted_motion_$_ids": [],
+            "assignment_candidate_$_ids": [],
+            "poll_voted_$_ids": [],
+            "option_$_ids": [],
+            "vote_$_ids": [],
+            "projection_$_ids": [],
+            "vote_delegated_vote_$_ids": [],
+            "vote_delegated_$_to_id": [],
+            "vote_delegations_$_from_ids": [],
+            "meeting_ids": [1],
+            **data,
+        }
+
+    def get_motion_data(self, obj_id: int, data: Dict[str, Any] = {}) -> Dict[str, Any]:
+        return {
+            "id": obj_id,
+            "meeting_id": 1,
+            "list_of_speakers_id": 1,
+            "state_id": 1,
+            "title": "bla",
+            "number": "1 - 1",
+            "number_value": 1,
+            "sequential_number": 2,
+            "text": "<p>l&ouml;mk</p>",
+            "amendment_paragraph_$": [],
+            "modified_final_version": "",
+            "reason": "",
+            "category_weight": 10000,
+            "state_extension": "<p>regeer</p>",
+            "recommendation_extension": None,
+            "sort_weight": 10000,
+            "created": 1584512346,
+            "last_modified": 1584512346,
+            "lead_motion_id": None,
+            "amendment_ids": [],
+            "sort_parent_id": None,
+            "sort_child_ids": [],
+            "origin_id": None,
+            "derived_motion_ids": [],
+            "all_origin_ids": [],
+            "all_derived_motion_ids": [],
+            "recommendation_id": None,
+            "recommendation_extension_reference_ids": [],
+            "referenced_in_motion_recommendation_extension_ids": [],
+            "category_id": None,
+            "block_id": None,
+            "submitter_ids": [],
+            "supporter_ids": [],
+            "poll_ids": [],
+            "option_ids": [],
+            "change_recommendation_ids": [],
+            "statute_paragraph_id": None,
+            "comment_ids": [],
+            "agenda_item_id": None,
+            "tag_ids": [],
+            "attachment_ids": [],
+            "projection_ids": [],
+            "personal_note_ids": [],
+            **data,
+        }
+
+    def test_initial_import_simple(self) -> None:
+        self.datastore.truncate_db()
+        request_data = self.create_request_data({})
+        response = self.request("organization.initial_import", request_data)
+        self.assert_status_code(response, 200)
+        self.assert_model_exists(
```

Comment:
check for all models from the request data automatically that they exist with the given content